iidm/iidm-serde/src/main/java/com/powsybl/iidm/serde/ImportOptions.java

```diff
@@ -79,4 +83,21 @@ public ImportOptions setMissingPermanentLimitPercentage(double missingPermanentL
     public double getMissingPermanentLimitPercentage() {
         return missingPermanentLimitPercentage;
     }
+
+    public ImportOptions setMinimalValidationLevel(String minimalValidationLevel) {
+        if (minimalValidationLevel != null) {
+            // no check?
```

Comment:
Maybe a check would be cleaner? But not mandatory, in any case, it should throw an exception if the value is not supported